src/smart-components/role/add-role/inventory-groups-role.js

```diff
@@ -133,78 +148,87 @@ const InventoryGroupsRole = (props) => {
     formOptions.change('inventory-group-permissions', groupsPermissionsDefinition);
   }, [state]);
 
-  const makeRow = (permissionId) => {
-    const options = Object.values(resourceTypes?.[permissionId] ?? {});
+  const makeRow = (permission, index) => {
```

Comment:
```suggestion
  const makeRow = (permissionId, index) => {
```
any reason for renaming the param? IMO `permissionId` is more descriptive since we are passing only ID

Reply:
When you look at the value per se, it's more of a `permissionType` rather than an unique ID for the individual permission.

All in all semantics rather than anything else, don't mind changing back